### PR TITLE
Adjust templates

### PR DESCRIPTION
**Pull request by LysandreJik**
Adjusting the issue and PR templates reflecting recent and upcoming changes in maintenance:

- Vision: cc @amyeroberts @qubvel 
- Speech: cc @ylacombe and @eustlb while Sanchit is off
- Pipelines: cc @Narsil and @Rocketknight1 (I can put myself as well here if you want @Rocketknight1 to help you shoulder some of the load)

### CODE REVIEW

**Review by amyeroberts — APPROVED** [review]
Thanks for updating!

**Review by Narsil — COMMENTED** [review]
LGTM :)

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -37,15 +37,15 @@ body:
...
           - graph models: @clefourrier
 
         Library:
 
           - flax: @sanchit-gandhi
           - generate: @zucchini-nlp (visual-language models) or @gante (all others)
-          - pipelines: @Narsil
+          - pipelines: @Rocketknight1
           - tensorflow: @gante and @Rocketknight1
           - tokenizers: @ArthurZucker
```
Comment (ArthurZucker) [inline .github/ISSUE_TEMPLATE/bug-report.yml:50]:
```suggestion
          - tokenizers: @ArthurZucker and @itazap
```

**Review by ArthurZucker — COMMENTED** [review]
thanks

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 1 above)
Comment (LysandreJik) [reply .github/ISSUE_TEMPLATE/bug-report.yml:50]: cc @itazap :)

**Comment by Rocketknight1** [comment]
@LysandreJik I'm happy to take pipelines! Should we list me for chat / chat templates as well?

**Comment by LysandreJik** [comment]
Yes, good idea! Will edit

**Pull request merged**